vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/TimelineItemFactory.kt

```diff
@@ -130,9 +137,18 @@ class TimelineItemFactory @Inject constructor(
                     EventType.CALL_ANSWER -> callItemFactory.create(params)
                     // Crypto
                     EventType.ENCRYPTED -> {
+                        val relationContent = event.getRelationContent()
                         if (event.root.isRedacted()) {
                             // Redacted event, let the MessageItemFactory handle it
                             messageItemFactory.create(params)
+                        } else if (relationContent?.type == RelationType.REFERENCE) {
+                            // Hide the decryption error for VoiceBroadcast chunks
+                            val startEvent = relationContent.eventId?.let { session.getRoom(event.roomId)?.getTimelineEvent(it) }
```

Comment:
At this moment we don't know if it's a voice broadcast, I think we should rename the variable like `relatedEvent`